core/src/main/java/lucee/transformer/bytecode/statement/tag/TagProperty.java

```diff
@@ -0,0 +1,195 @@
+
+import org.objectweb.asm.Type;
+import org.objectweb.asm.commons.GeneratorAdapter;
+import org.objectweb.asm.commons.Method;
+
+import lucee.transformer.Factory;
+import lucee.transformer.Position;
+import lucee.transformer.TransformerException;
+import lucee.transformer.bytecode.BytecodeContext;
+import lucee.transformer.bytecode.statement.FlowControlFinal;
+import lucee.transformer.bytecode.util.ASMConstants;
+import lucee.transformer.bytecode.util.Types;
+import lucee.transformer.statement.tag.Attribute;
+import lucee.transformer.statement.tag.Tag;
+
+import java.util.Iterator;
+
+/**
+ * Optimized bytecode generation for cfproperty tags.
+ * Bypasses tag lifecycle overhead by calling ComponentUtil.registerProperty directly.
+ */
+public final class TagProperty extends TagBase {
+
+	public TagProperty(Factory f, Position start, Position end) {
+		super(f, start, end);
+	}
+
+	@Override
+	public FlowControlFinal getFlowControlFinal() {
+		return null;
+	}
+
+	@Override
+	public void _writeOut(BytecodeContext bc) throws TransformerException {
+		final GeneratorAdapter adapter = bc.getAdapter();
+		Tag tag = (Tag) this;
+
+		System.err.println("[DEBUG-PROPERTY] TagProperty._writeOut called for optimized property processing");
+
+		bc.visitLine(tag.getStart());
+
+		// Extract property attributes
+		String name = null;
+		String type = null;
+		Object defaultValue = null;
+		String access = null;
+		String hint = null;
+		String displayname = null;
+		boolean required = false;
+		boolean setter = true;
+		boolean getter = true;
+
+		// Parse tag attributes
+		Iterator<Attribute> it = tag.getAttributes().values().iterator();
+		while (it.hasNext()) {
+			Attribute attr = it.next();
+			String attrName = attr.getName().toLowerCase();
+
+			if ("name".equals(attrName)) {
+				name = getLiteralString(attr);
+			}
+			else if ("type".equals(attrName)) {
+				type = getLiteralString(attr);
+			}
+			else if ("default".equals(attrName)) {
+				// Handle default value - simplified for now
+				defaultValue = getLiteralString(attr);
+			}
+			else if ("access".equals(attrName)) {
+				access = getLiteralString(attr);
+			}
+			else if ("hint".equals(attrName)) {
+				hint = getLiteralString(attr);
+			}
+			else if ("displayname".equals(attrName)) {
+				displayname = getLiteralString(attr);
+			}
+			else if ("required".equals(attrName)) {
+				String val = getLiteralString(attr);
+				required = "yes".equalsIgnoreCase(val) || "true".equalsIgnoreCase(val);
+			}
+			else if ("setter".equals(attrName)) {
+				String val = getLiteralString(attr);
+				setter = !"no".equalsIgnoreCase(val) && !"false".equalsIgnoreCase(val);
+			}
+			else if ("getter".equals(attrName)) {
+				String val = getLiteralString(attr);
+				getter = !"no".equalsIgnoreCase(val) && !"false".equalsIgnoreCase(val);
+			}
+		}
+
+		System.err.println("[DEBUG-PROPERTY] Generating optimized call: ComponentUtil.registerProperty(pc, name=\"" + name + "\", type=\"" + type + "\")");
```

Comment:
please remove this.
Tip: use instead print.e or print.o, it is not only better for showing cfml specific tyes, you can lso simply remove the "extends aprint" in that class and you see where it is used and clean up add the end of development.